DigitalLearningSolutions.Web/Controllers/HomeController.cs

```diff
@@ -69,7 +69,7 @@ public IActionResult LearningContent()
                 MiniHubNavigationModel = new MiniHubNavigationModel(LandingPageMiniHubName, sections, 2),
                 UserIsLoggedIn = User.Identity.IsAuthenticated,
                 CurrentSiteBaseUrl = configuration.GetCurrentSystemBaseUrl(),
-                LearningContents = learningContents,
+                LearningContentItems = learningContents,

```

Comment:
Minor, but can we call this variable "learningContents" also "learningContentItems" to match the Model property, or could call it "publicBrands" to describe what it is more specifically